packages/twenty-front/src/modules/ui/input/components/Select.tsx

```diff
@@ -93,11 +93,29 @@ export const Select = <Value extends SelectValue>({
   const selectContainerRef = useRef<HTMLDivElement>(null);
 
   const [searchInputValue, setSearchInputValue] = useState('');
+  const getSelectedOption = () => {
+    const fromMatchingOption = options.find(
+      ({ value: optionValue }) => optionValue === value,
+    );
 
-  const selectedOption =
-    options.find(({ value: key }) => key === value) ||
-    emptyOption ||
-    options[0];
+    if (fromMatchingOption !== undefined) {
```

Comment:
use `!isDefined()`

Reply:
Would involve a regression,`fromMatchingOption` is nullable

Reply:
```ts

export type SelectOption<
  Value extends string | number | boolean | null = string,
> = {
  Icon?: IconComponent | null;
  label: string;
  value: Value;
  disabled?: boolean;
  color?: ThemeColor | 'transparent';
};

```

Reply:
Whether it's just `null` or just `undefined` doesn't call for avoiding `isDefined`, on the contrary, using `isDefined` prevents introducing future bugs if someone changes the type to `undefined | null` (for example because a caller of Select wants to pass null and that it is easier to change the API of Select. 

This can happen a lot, handling just undefined or just null leaves the door open for bugs, it's an open-ended bug trap, while using isDefined deterministically closes the door to any bug that would involve a `null` value not being caught by `=== undefined` or the opposite, thus avoiding altogether the need to maintain null or undefined specific code paths, which in 99.999999% of the cases brings no benefit being treated separately.

Reply:
Also I hope that we agree on that very soon because that's how we handle nullable values since 2 years, so better not introducing and favoring another way to work with nullable now.

Reply:
To go in your direction, we could separate here indeed the option.value treatment between null and undefined, but this is not required here and the code that was checking for undefined wasn't checking option.value but option object, which shouldn't be treated as undefined only but nullable with `isDefined`

Reply:
And you're right for empty option array, but that's again another use case than null | undefined. 

See this issue : https://github.com/twentyhq/core-team-issues/issues/1826